lib/workers/branch/schedule.ts

```diff
@@ -123,13 +165,22 @@ export function isScheduledNow(config: RenovateConfig): boolean {
 
   // We run if any schedule matches
   const isWithinSchedule = configSchedule.some((scheduleText) => {
-    const massagedText = scheduleMappings[scheduleText] || scheduleText;
-    const parsedSchedule = later.parse.text(fixShortHours(massagedText));
-    logger.debug({ parsedSchedule }, `Checking schedule "${scheduleText}"`);
+    if (scheduleText.indexOf(minutesChar) === 0) {
```

Comment:
I'd prefer if there was a "is this valid cron?" as the first check and then verifying the minutes is *. Also we should have config validation for this where we reject any valid cron which doesn't have * minutes